apps/ledger-live-mobile/src/components/Carousel/index.tsx

```diff
@@ -64,18 +64,21 @@ const Carousel = ({ cardsVisibility }: Props) => {
 
   const slides = useMemo(
     () =>
-      getDefaultSlides().filter((slide: any) => {
-        if (!cardsVisibility[slide.id]) {
-          return false;
-        }
-        if (slide.start && slide.start > new Date()) {
-          return false;
-        }
-        if (slide.end && slide.end < new Date()) {
-          return false;
-        }
-        return true;
-      }),
+      getDefaultSlides().filter(
+        (slide: { id: string; Component: () => JSX.Element }) => {
+          if (!cardsVisibility[slide.id]) {
+            return false;
+          }
+          // FIXME: start / end field do not seem to be valid
```

Comment:
change of logic, who should be involved at fixing this?

Reply:
just a guess but @LedgerHQ/live-hub  seem to be the proper team

Reply:
the start/end  is probably a part from the old carousel that we just copy pasted while doing v3.
I will check if there is any data in slides with these properties, but if i recall, it was empty/deprecated

Reply:
done, removed